docs/reference/filters.md

```diff
@@ -1748,6 +1748,106 @@ foo: Path("/baz")
 Configures rate limit of 100 requests per second for each `backend1` and `backend2` and responds
 with `429 Too Many Requests` when limit is reached.
 
+## clusterLeakyBucketRatelimit
+
+Implements leaky bucket rate limit algorithm that uses Redis as a storage.
+Requires command line flags `-enable-ratelimits`, `-enable-swarm` and `-swarm-redis-urls` to be set.
```

Comment:
Maybe we should add also install instruction in https://opensource.zalando.com/skipper/tutorials/ratelimit/#redis-based-cluster-ratelimits , but a hint here is also fine.